microsoft-edge/visual-studio-code/debugger-for-edge.md

```diff
@@ -1,17 +1,23 @@
 ---
-description: How to debug Microsoft Edge from Visual Studio Code
-title: Debug Microsoft Edge from Visual Studio Code
+description: You can debug Microsoft Edge from Visual Studio Code using the built in debugger
+title: Debug Microsoft Edge in Visual Studio Code
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 07/26/2021
+ms.date: 11/10/2021
 ms.topic: article
 ms.prod: microsoft-edge
 keywords: microsoft edge, web development, f12 tools, devtools, vs code, visual studio code, debugger
 ---
-# Debug Microsoft Edge from Visual Studio Code
+# Debug Microsoft Edge in Visual Studio Code
 
-[Visual Studio Code](https://code.visualstudio.com) includes a built-in debugger for Microsoft Edge which can launch the browser or attach to an already running browser.
+[Visual Studio Code](https://code.visualstudio.com) includes a built-in debugger for Microsoft Edge, which can launch the browser or attach to an already running browser.
 
-This built-in debugger lets you debug your front-end JavaScript code line-by-line and see `console.log()` statements directly from [Visual Studio Code](https://code.visualstudio.com).
+This built-in debugger lets you debug your front-end JavaScript code line-by-line and see `console.log()` statements directly from Visual Studio Code.
 
-For more information, navigate to [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging).
+
+
+For more information, see [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging).
+
+You can also inspect HTML and CSS using DevTools from within VS Code. To learn more, see [Browser debugging with Microsoft Edge Devtools integration in VS Code](/microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension#browser-debugging-with-microsoft-edge-devtools-integration-in-visual-studio-code).
+
+Send your feedback by [filing an issue](https://github.com/microsoft/vscode-js-debug/issues/new) in the [vscode-js-debug repository](https://github.com/microsoft/vscode-js-debug).
```

Comment:
Possibly move this paragraph above the "You can also" paragraph.  The structure of the article right now is:
1. Tool A.
2. Tool B.
3. How to give feedback (about tool A).
And/or, to disambiguate which tool the feedback is about, change "Send your feedback..." to "Send your feedback about (tool A)..."